themis/regulations/apple-crypto-regulations.md

```diff
@@ -28,7 +28,18 @@ Read more in the official [Apple export compliance overview](https://help.apple.
 and [BIS guidelines](https://bis.doc.gov/index.php/policy-guidance/encryption/1-encryption-items-not-subject-to-the-ear)
 (you might need to use a VPN to access it).
 
-#### Submitting an annual self-classification report to Apple
+
+# US crypto export regulations for Themis when submitting to Google Play
+
+If your application uses Themis and you want to submit it to the Google Play,
+you are required to do the following:
+
+1. Send an annual (year-end) self-classification report to the US government
+    to comply with the [encryption export regulations](https://developer.apple.com/documentation/security/complying_with_encryption_export_regulations).
```

Comment:
I don't feel it's right to link to Apple docs in the Google Play section. Since Google does not seem to provide more details on the matter other than the Google Play guidelines linked below, I think we can leave only that link.

```suggestion
 1. Send an annual (year-end) self-classification report to the US government
    to comply with the encryption export regulations.
```

Also, I have an OCD for indentation alignment.